testing/pytest-cov/APKBUILD

```diff
@@ -0,0 +1,51 @@
+# Contributor: TBK <alpine@jjtc.eu>
+# Maintainer: TBK <alpine@jjtc.eu>
+pkgname=pytest-cov
+_pkgname=${pkgname#py-}
+pkgver=2.5.1
+pkgrel=0
+pkgdesc="Pytest plugin for measuring coverage."
+url="https://github.com/pytest-dev/pytest-cov"
+arch="noarch"
+license="MIT"
+depends="pytest py-coverage"
+makedepends="python2-dev python3-dev py-setuptools"
+subpackages="py3-${pkgname/py-/}:_py3 py2-${pkgname/py-/}:_py2"
+source="$pkgname-$pkgver.tar.gz::https://github.com/pytest-dev/pytest-cov/archive/v$pkgver.tar.gz"
+builddir="$srcdir/$_pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
+	python2 setup.py build
+	python3 setup.py build
+}
+
+check() {
+	cd "$builddir"
+	python2 setup.py test
+	python3 setup.py test
+}
+
+package() {
+	mkdir -p "$pkgdir"
+}
+
+_py() {
+	local python=$1
+	pkgdesc="$pkgdesc - $python"
+	install_if="$pkgname=$pkgver-r$pkgrel $python"
+
+	cd "$builddir"
+	$python setup.py install --prefix=/usr --root="$subpkgdir"
+}
+
+_py2() {
+	replaces="$pkgname"
```

Comment:
Did you check the [wiki page on python aports](https://wiki.alpinelinux.org/wiki/APKBUILD_examples:Python#Multiversion_package_without_executables)?

It states that this line should be removed when writing a new aport:

```shell
replaces="$pkgname"  ## remove if you're writing a new aport
```